docs/applications/bitcoin-core.md

```diff
@@ -46,81 +50,139 @@ proxy=127.0.0.1:9050
+reindex-chainstate=0
 ```
 
 Some notes about this specific configuration:
 
-* **Application options**
-  * `testnet`: the build script defaults to building a testnet node, but can be configured to run on mainnet (commenting out this line):
-    * by specifying the corresponding build parameter in [`build.conf`](../../armbian/base/build/build.conf)
-    * or by running the command `bbb-config.sh set bitcoin_network mainnet` manually on the BitBox Base (see [OS/Helper Scripts](../os/helper-scripts.md)).
+* **Network options**
+  * `mainnet`/`testnet`: the build script defaults to building a mainnet node, but can be reconfigured by:
+    * specifying the corresponding build parameter in [`build.conf`](../../armbian/base/build/build.conf)
+    * or running the command `bbb-config.sh set bitcoin_network mainnet` manually on the BitBox Base (see [Operating System/Helper Scripts](../os/helper-scripts.md)).
+
+* **Server options**
   * `server`: enables the RPC interface
   * `listen`: accept connections from other nodes
   * `listenonion`: create a Tor hidden service and accept incoming connections from other nodes on that address
-  * `daemon`: run bitcoind as a server in the background
-  * `txindex`: a full transaction index is not necessary, as we use electrs - which has its own indices - to serve transaction information.
-  * `prune`: both c-lightning as well as electrs do not support pruned Bitcoin nodes at the moment.
+  * `daemon`: to caputre all log output with the `printtoconsole` option, bitcoind must not run as a daemon
+  * `txindex`: a full transaction index is not necessary, as we use electrs with its own indices to serve transaction information
+  * `prune`: both c-lightning as well as electrs do not fully support pruned Bitcoin nodes at the moment
   * `disablewallet`: wallet functionality of bitcoind is not used and therefore disabled
-  * `pid`: this file contains the process id of bitcoind
-  * `rpccookiefile`: other applications using the bitcoind RPC authenticate themselves by reading the access credentials from the `.cookie` file. To avoid using different storage locations when running Bitcoin mainnet or testnet, the location is explicitly specified.
-  * `sysparms`: other system users that are a member of the `bitcoin` group need to have read-only access to the `.cookie` file. Using this option, bitcoind creates new files with system default permissions instead of the umask 0777.
-  * `rpcconnect`: as the bitcoind rpc port is sensitive, it listens for local connections only.
+
+* **RPC options**
+  * `rpconnect`: specifying `127.0.0.1`, the bitcoind api listens only for local connections 
+  * `rpcport`: the bitcoind api always listens on port `8334`, both for mainnet and testnet 
+  * `rpcauth`: authentication to bitcoind api uses the `rpcauth` method, with clients using static `rpcuser` and `rpcpassword` values. Credentials are created using an [adapted version](TODO) of the Bitcoin Core [rpcauth.py](TODO) Python script
```

Comment:
Here are two TODOs that might have been forgotten, if not just ignore.

Reply:
Thanks, fixed all three TODOs